core/data.go

```diff
@@ -625,3 +626,8 @@ type BlobVersionParameters struct {
 	MaxNumOperators uint32
 	NumChunks       uint32
 }
+
+func (ar *ReservedPayment) IsActive() bool {
+	now := uint64(time.Now().Unix())
```

Comment:
do we have unit tests for active/inactive reservations? 
would be useful to take this timestamp as input for testing